assets/apps/customizer-controls/src/builder-instructions/Instructions.tsx

```diff
@@ -66,7 +71,28 @@ const Instructions: React.FC<Props> = ({ control }) => {
 									>
 										<span className={`dashicons ${icon}`} />
 										{label}
+										{url && badge && (
+											<span className="quick-links-badge">
+												{badge}
+											</span>
+										)}
 									</Button>
+									{url && upsellDescription && (
+										<div className="quick-links-upsell">
+											<p className="quick-links-description">
+												<div
+													dangerouslySetInnerHTML={{
+														// eslint-disable-next-line @wordpress/valid-sprintf
+														__html: sprintf(
+															upsellDescription,
+															`<a href="${url}" target="_blank" rel="noopener noreferrer">`,
+															'</a>'
+														),
+													}}
+												/>
+											</p>
+										</div>
+									)}
```

Comment:
@Soare-Robert-Daniel Let's have the `<a>` tag in the localization itself. If we add another upsell description without a link in the future, this is redundant. As long as we're already doing a dangerouslySetHTML here, I see no point of doing the `sprintf` here instead of in PHP directly.